File: jobs/2026-06-11__15-58-32/agentkit_list_github_repos_ts__ENbo7jP/artifacts/user/myproject/explore.js
import { Scalekit } from '@scalekit-sdk/node';
console.log("Starting...");
console.log("ENV URL:", process.env.SCALEKIT_ENV_URL);
async function main() {
    const scalekit = new Scalekit(process.env.SCALEKIT_ENV_URL, process.env.SCALEKIT_CLIENT_ID, process.env.SCALEKIT_CLIENT_SECRET);
    const identifier = 'zealt-user01';
    const connectionName = 'github-test';
    try {
        console.log("Executing tool...");
        const result = await scalekit.actions.executeTool({
            identifier,
            connector: connectionName,
            toolName: 'github_list_repos_for_authenticated_user',
            toolInput: { per_page: 100 }
        });
        console.log("Result:", JSON.stringify(result, null, 2));
    }
    catch (error) {
        console.error("Error:", error);
    }
}
main().then(() => console.log("Done")).catch(e => console.error(e));

File: jobs/2026-06-11__15-58-32/agentkit_create_github_repo_py__jkYbfEv/artifacts/user/myproject/explore.py
import os
import json
from scalekit import ScalekitClient

def main():
    client = ScalekitClient(
        os.environ['SCALEKIT_ENV_URL'],
        os.environ['SCALEKIT_CLIENT_ID'],
        os.environ['SCALEKIT_CLIENT_SECRET']
    )
    res = client.tools.execute_tool(
        tool_name="githubmcp_create_repository",
        identifier="zealt-user01",
        connection_name="github-test",
        params={
            "name": "agentkit-repo-test-999",
            "private": False,
            "autoInit": True
        }
    )
    print(dir(res[0]))
    print(res[0].data)

if __name__ == "__main__":
    main()
